README.md

```diff
@@ -522,6 +586,62 @@ This changes if one of the primary key columns is an identity column though. In
+
+1. To enable change tracking on the table, run
+
+    ```sql
+    ALTER TABLE dbo.Employees
+    ENABLE CHANGE_TRACKING
+    WITH (TRACK_COLUMNS_UPDATED = ON)
+    ```
+
+    The `TRACK_COLUMNS_UPDATED` feature being enabled means that the change tracking table also stores information about what columns where updated in the case of an `UPDATE`. Currently, the trigger does not make use of this additional metadata, though that functionality could be added in the future. More information about this command is provided [here](https://docs.microsoft.com/sql/relational-databases/track-changes/enable-and-disable-change-tracking-sql-server?view=sql-server-ver15#enable-change-tracking-for-a-table).
+
+    The trigger needs to have read access to the table being monitored for changes as well as to the change tracking system tables. It also needs write access to an `az_func` schema within the database, where it will create additional worker tables to process the changes. Each user table will thus have an associated change tracking table and worker table. The worker table will contain roughly as many rows as the change tracking table, and will be cleaned up approximately as often as the change table.
+
+#### Trigger Samples
+The trigger takes two arguments
+
+- **TableName**: Passed as a constructor argument to the binding. Represents the name of the table to be monitored for changes.
+- **ConnectionStringSetting**: Specifies the name of the app setting that contains the SQL connection string used to connect to a database. The connection string must follow the format specified [here](https://docs.microsoft.com/dotnet/api/microsoft.data.sqlclient.sqlconnection.connectionstring?view=sqlclient-dotnet-core-2.0).
+
+The following are valid binding types for trigger
+
+- **IReadOnlyList<SqlChange\<T\>>**: Each element is a `SqlChange`, which stores change metadata about a modified row in the user table as well as the row itself. In the case that the row was deleted, only the primary key values of the row are populated. The user table row is represented by `T`, where `T` is a user-defined POCO, or Plain Old C# Object. `T` should follow the structure of a row in the queried table. See the [Query String](#query-string) section for an example of what `T` should look like. The two fields of a `SqlChange` are the `Item` field of type `T` which stores the row, and the `Operation` field of type `SqlChangeOperation` which indicates the type of operaton done to the row (either an insert, update, or delete).
```

Comment:
Why are only the primary key values populated when an item is deleted?

Reply:
This is because of the change tracking table, which stores only primary key values.